closed/OpenJ9.gmk

```diff
@@ -179,6 +179,14 @@ $(foreach subdir, j9vm server, \
 		$(subdir)/$(call SHARED_LIBRARY,jvm), \
 		redirector/$(call SHARED_LIBRARY,jvm_jdk11)))
 
+# jsig
+$(call openj9_add_jdk_special, \
+                server/$(call SHARED_LIBRARY,jsig), \
```

Comment:
This should add jsig to the `j9vm` subdirectory (not just the `server` subdirectory.

Please also respect the existing indentation style of this file.

Reply:
indentation corrected
subdirs match where libjsig.so was before